src/app/modules/config/config.events.ts

```diff
@@ -0,0 +1,9 @@
+export const ConfigInitEvent = Symbol('Config Init');
+
+export type ConfigInitPayload = {
+    aboutPage: any;
```

Comment:
please create interface for aboutPage, this is quite complex object

Reply:
created type AboutPage